types/stack/index.d.ts

```diff
@@ -92,4 +93,7 @@ export interface Stack extends SystemFields {
     unShare(email: string): Promise<Response>
     role(): Roles
     role(uid: string): Role
+
+    taxonomy(): Taxonomy
+    taxonomies(uid: string): Taxonomies
```

Comment:
Change this to taxonomy @harshithad0703 
Even for collection we use the same function name